buf/registry/module/v1beta1/commit_service.proto

```diff
@@ -181,6 +181,8 @@ message CreateCommitsRequest {
   //
   // If there are already VCSCommits on the associated Module with a given hash, this
   // will result in an error. Otherwise, a new VCSCommit is created.
+  //
+  // This operation is atomic. Either all Commits are created, or an error happens and none of them are created.
```

Comment:
I think this comment is on the wrong spot - this comment is on the associated VCS commit field, I think you mean to say that all Commits in the workspace, as WELL AS all associated VCS commits, are created atomically, and this comment should likely be on the RPC or Request level.

Reply:
Good catch. Since RPCs already have doc strings and the request objects don't, I moved the docs to the RPC for this one. It does beg the question on whether the other comments should get moved to the RPC level as well.

Reply:
I moved all docs to be on the RPC method since that seems to make more sense and is more consistent.